src/arkode/arkode.c

```diff
@@ -1278,13 +1278,16 @@ int arkInit(ARKodeMem ark_mem, realtype t0, N_Vector y0,
 
   /* All allocation and error checking is complete at this point */
 
-  /* Copy the input parameters into ARKode state */
+  /* Copy the input parameters into ARKODE state */
   ark_mem->tcur = t0;
   ark_mem->tn   = t0;
 
   /* Initialize yn */
   N_VScale(ONE, y0, ark_mem->yn);
 
+  /* Clear any previous 'tstop' */
+  ark_mem->tstopset = SUNFALSE;
```

Comment:
I think the documentation for the `ReInit` and `Reset` functions needs to be updated to note that after a reinitialization or reset any previously set stop time will be disabled.